### PR TITLE
Create archive-node-nodereal

### PR DESCRIPTION
**Pull request by STdevK**
NodeReal support Archive Node on Polygon mainnet

### CODE REVIEW

File: docs/operate/archive-node-nodereal [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+image: https://wiki.polygon.technology/img/polygon-logo.png
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Get access to an Archive Node with NodeReal
+
+New to Web3 development? Don't worry, you won't have to spin up your own full node on Polygon – NodeReal has got you covered!
```
Comment (DannyS03) [inline docs/operate/archive-node-nodereal:22]:
```suggestion
If you're new to Web3 development, [NodeReal](https://nodereal.io/) simplifies the process of setting up a full node on Polygon.
```

File: docs/operate/archive-node-nodereal [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+Most developers opt for a node provider – a quick and easy third-party service that handles requests and responses – because it's the fastest route to getting started on Polygon. NodeReal can help you get up and running!
```
Comment (DannyS03) [inline docs/operate/archive-node-nodereal:24]:
```suggestion
:::tip Simplify Polygon node usage with NodeReal and its quality assurance capabilities

Many developers prefer to use a node provider when getting started on Polygon, or any chain for that matter, as it offers a quick and easy way to handle requests and responses. NodeReal offers a range of features for developers looking for a node provider on the Polygon network. These features include a Standard Interface that simplifies node usage, as well as reliability and scalability for optimal performance. Additionally, NodeReal offers Archive Data for secure data storage, and an API Marketplace that provides additional resources for developers looking to build on Polygon.

```

File: docs/operate/archive-node-nodereal [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+
+:::tip Quality Assurance
+
+NodeReal's Standard Interface, Reliability, Scalability, Archive Data, and [API Marketplace](https://nodereal.io/api-marketplace) make it a good choice for new Polygon network developers looking for a node provider.
```
Comment (DannyS03) [inline docs/operate/archive-node-nodereal:28]:
```suggestion
```

File: docs/operate/archive-node-nodereal [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Get access to an Archive Node with NodeReal
+
+New to Web3 development? Don't worry, you won't have to spin up your own full node on Polygon – NodeReal has got you covered!
+
+Most developers opt for a node provider – a quick and easy third-party service that handles requests and responses – because it's the fastest route to getting started on Polygon. NodeReal can help you get up and running!
+
+:::tip Quality Assurance
```
Comment (DannyS03) [inline docs/operate/archive-node-nodereal:26]:
```suggestion
```

File: docs/operate/archive-node-nodereal [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+
+NodeReal's Standard Interface, Reliability, Scalability, Archive Data, and [API Marketplace](https://nodereal.io/api-marketplace) make it a good choice for new Polygon network developers looking for a node provider.
+
+:::
```
Comment (DannyS03) [inline docs/operate/archive-node-nodereal:30]:
```suggestion
```

File: docs/operate/archive-node-nodereal [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+# Get access to an Archive Node with NodeReal
+
+New to Web3 development? Don't worry, you won't have to spin up your own full node on Polygon – NodeReal has got you covered!
+
+Most developers opt for a node provider – a quick and easy third-party service that handles requests and responses – because it's the fastest route to getting started on Polygon. NodeReal can help you get up and running!
+
+:::tip Quality Assurance
+
+NodeReal's Standard Interface, Reliability, Scalability, Archive Data, and [API Marketplace](https://nodereal.io/api-marketplace) make it a good choice for new Polygon network developers looking for a node provider.
+
```
Comment (DannyS03) [inline docs/operate/archive-node-nodereal:29]:
```suggestion
```

File: docs/operate/archive-node-nodereal [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Get access to an Archive Node with NodeReal
+
+New to Web3 development? Don't worry, you won't have to spin up your own full node on Polygon – NodeReal has got you covered!
+
+Most developers opt for a node provider – a quick and easy third-party service that handles requests and responses – because it's the fastest route to getting started on Polygon. NodeReal can help you get up and running!
+
```
Comment (DannyS03) [inline docs/operate/archive-node-nodereal:25]:
```suggestion
:::
```

File: docs/operate/archive-node-nodereal [hunk 8]
```diff
@@ -0,0 +1,166 @@
...
+:::tip Quality Assurance
+
+NodeReal's Standard Interface, Reliability, Scalability, Archive Data, and [API Marketplace](https://nodereal.io/api-marketplace) make it a good choice for new Polygon network developers looking for a node provider.
+
+:::
+
+## Send Your First Blockchain API Request
+
+We assumes you have signed up for a [MegaNode account](https://nodereal.io/meganode) and can access the [NodeReal Dashboard](https://dashboard.nodereal.io). “Where to sign up for a MegaNode account?” you asked. We got your back and here’s the [guide](https://docs.nodereal.io/docs/getting-started).
```
Comment (DannyS03) [inline docs/operate/archive-node-nodereal:34]:
```suggestion
If you haven't signed up for a MegaNode account yet, you can find a step-by-step guide to getting started [here](https://docs.nodereal.io/docs/getting-started). Once you have a MegaNode account and access to the NodeReal Dashboard, you're ready to proceed.
```

File: docs/operate/archive-node-nodereal [hunk 9]
```diff
@@ -0,0 +1,166 @@
...
+
+You can retrieve the Polygon **Https URL** from your dashboard. 
+
+![img](https://files.readme.io/94f4b79-Screen_Shot_2023-01-16_at_20.24.34.png)
+
+If you encounter issue, do reach out to us at (Discord)[https://discord.com/invite/nodereal].
+
+### Install dependency for `index.js` 
+
+npm
```
Comment (DannyS03) [inline docs/operate/archive-node-nodereal:141]: are these here intentionally? (the npm and yarn keywords above the code block)

File: docs/operate/archive-node-nodereal [hunk 10]
```diff
@@ -0,0 +1,166 @@
...
+
+### Response in Your Terminal
+
+You should be able to see the latest block number and block details on your terminal.
+
+```bash
+Latest block number: 40302452
+Latest block details: {.............}
+```
+
```
Comment (DannyS03) [inline docs/operate/archive-node-nodereal:165]: May we please add a "testing" or "troubleshooting" section before completion? If there is a guide that already exists, we can link to it with a statement or two before referencing it to introduce the section.

File: docs/operate/archive-node-nodereal [hunk 11]
```diff
@@ -0,0 +1,166 @@
...
+:::
+
+## Send Your First Blockchain API Request
+
+We assumes you have signed up for a [MegaNode account](https://nodereal.io/meganode) and can access the [NodeReal Dashboard](https://dashboard.nodereal.io). “Where to sign up for a MegaNode account?” you asked. We got your back and here’s the [guide](https://docs.nodereal.io/docs/getting-started).
+
+### Create Your First API Key
+
+:::Fun Fact: NodeReal Upgrade from "APP" to "API Key"
```
Comment (DannyS03) [inline docs/operate/archive-node-nodereal:38]: this won't work as an admonition unfortunately.

I'd advice using ::tip and something along the lines of

```
:::tip One API key for all chains

[..]

:::

I'd also move this until after the step where a user creates an API key.

File: docs/operate/archive-node-nodereal [hunk 12]
```diff
@@ -0,0 +1,166 @@
...
+
+With only one API Key, you can request all the chains and networks we support on NodeReal.
+
+:::
+
+![img](https://files.readme.io/9352cd2-Screen_Shot_2023-01-10_at_19.26.00.png)
+
+Before you start sending RPC requests on MegaNode, you will need to create an API Key first. API Key in MegaNode is a project-based unit used to manage your API requests. You need to input a name for your API Key and leave any notes you prefer for future reference.
```
Comment (DannyS03) [inline docs/operate/archive-node-nodereal:46]:
```suggestion
To start sending RPC requests on MegaNode, you'll need to create an API Key. This project-based unit is used to manage your API requests, and requires you to input a name for the key and leave any notes you prefer for future reference. Here's how to get started:
```

**Review by DannyS03 — CHANGES_REQUESTED** [review]
thank you for the contribution @STdevK. I added some suggestions. In addition:

- Let's number each step and aim to make each step direct and actionable.
- I'd put important provider information about NodeReal at the beginning of the associated section, such as the note on API key pricing.

**Comment by ashucoder9** [comment]
Hey @STdevK, a lot of changes requested are not addressed yet. Can you please take a look again and fix them before review?

File: docs/operate/archive-node-nodereal (hunk 11 above)
Comment (STdevK) [reply docs/operate/archive-node-nodereal]: Thank you for highlighting, I've made the changes and shift it after user creates an API key

**Review by DannyS03 — CHANGES_REQUESTED** [review]
Thanks for the updates @darron.

Added a few more comments, mainly nits.

**Review by DannyS03 — APPROVED** [review]

**Pull request merged**